indigo_app/urls.py

```diff
@@ -48,6 +48,7 @@
     path('places/<str:place>/localities', places.PlaceLocalitiesView.as_view(), name='place_localities'),
 
     path('places/<str:place>/tasks', tasks.TaskListView.as_view(), name='tasks'),
+    path('places/<str:place>/tasks/facets', tasks.TaskFacetsView.as_view(), name='tasks_facets'),
```

Comment:
Pls apply 30 seconds of caching, like we do for `places/<str:place>/works/facets`